builders/interact/ethereum-api/dev-env/remix.md

```diff
@@ -0,0 +1,99 @@
+
+
+## Getting Started with Remix {: #getting-started-with-remix } 
+
+Now, you can head to [Remix](https://remix.ethereum.org/){target=_blank} to get started. In the main screen, under **Featured Plugins**, select **SOLIDITY** to configure Remix for Solidity development, then navigate to the **File Explorers** view.
+
+![File explorer](/images/builders/interact/ethereum-api/dev-environments/remix/remix-1.png)
+
+You will create a new file to save the Solidity smart contract. Press the **Create New File** button on the left hand side of **File Explorers** and enter the name `MyToken.sol` in the pop-up.
+
+![Create a new file for your Solidity contract](/images/builders/interact/ethereum-api/dev-environments/remix/remix-2.png)
+
+Next, paste the following smart contract into the editor tab:
+
+```solidity
+--8<-- 'code/ethereum-api/remix/erc20.sol'
+```
+
+![Paste the contract into the editor](/images/builders/interact/ethereum-api/dev-environments/remix/remix-3.png)
+
+This is a simple ERC-20 contract based on the [current OpenZeppelin ERC-20 template](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/ERC20.sol){target=_blank}. It creates `MyToken` with symbol `MYTOK` and mints the entirety of the initial supply to the creator of the contract.
+
+Now, navigate to the **Solidity compiler** sidebar option and press the **Compile MyToken.sol** button.
+
+![Compile MyToken.sol](/images/builders/interact/ethereum-api/dev-environments/remix/remix-4.png)
+
+You will see Remix download all of the OpenZeppelin dependencies and compile the contract.
+
+## Deploying a Contract to your ContainerChain Using Remix {: #deploying-a-contract-to-your-containerchain-using-remix }
+
+Now you can deploy the contract by navigating to the **Deployment** sidebar option. You need to change the topmost **ENVIRONMENT** dropdown from **JavaScript VM** to **Injected Web3**. This tells Remix to use the MetaMask injected provider, which will point it to your EVM ContainerChain, so long as the selected network in your MetaMask is your EVM ContainerChain. If you need to change your network in MetaMask, you can easily do so, and Remix will update your account balances to reflecting the network change. 
+
+As soon as you select **Injected Web3**, you will be prompted to allow Remix to connect to your MetaMask account.
+
+![Replace](/images/builders/interact/ethereum-api/dev-environments/remix/remix-5.png)
+
+Press **Next** in MetaMask to allow Remix to access the selected account.
+
+Back on Remix, you should see that the account you wish to use for deployment is now managed by MetaMask. Next to the **Deploy** button, specify an initial supply of 8M tokens. Since this contract uses the default of 18 decimals, the value to put in the box is `8000000000000000000000000`.
```

Comment:
```suggestion
Back on Remix, you should see the account you wish to use for deployment is now managed by MetaMask. Next to the **Deploy** button, specify an initial supply of 8 million tokens. Since this contract uses the default of 18 decimals, the value to put in the box is `8000000000000000000000000`.
```